src/containers/guest-home-page/login-form/LoginForm.js

```diff
@@ -12,12 +14,26 @@ import { style } from './login-form.style'
 const LoginForm = ({ handleSubmit, handleChange, handleBlur, data, errors }) => {
   const { inputVisibility: passwordVisibility, showInputText: showPassword } = useInputVisibility(errors.password)
   const { setModal } = useContext(ModalContext)
+  const { loading } = useSelector((state) => state.appMain)
   const { t } = useTranslation()
 
   const openForgotPassword = () => {
     setModal(<ForgotPassword />)
   }
 
+  const loginButton = loading ? (
+    <Box sx={ { minWidth: '100px', display: 'flex', justifyContent: 'center' } }>
```

Comment:
I think, it would be better to move **{ minWidth: '100px', display: 'flex', justifyContent: 'center' }** to style

Reply:
changed